funcx_sdk/funcx/serialize/concretes.py

```diff
@@ -143,6 +225,7 @@ def deserialize(self, payload):
     ]
```

Comment:
Without adding `CombinedCode`, how will `FuncXSerializer` know how to deserialize messages in this format?